src/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './base';
 export * from './coins';
 export * from './networks';
 export * from './errors';
+export * from './ofc';
```

Comment:
I think this can just be. `export { OfcCoin } from './ofc';`.

No need to export the other stuff in there.